circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  override:
+    - printf '%s\n' color=false progress=false >~/.npmrc
+    - make setup
+
+machine:
+  node:
+    version: 4
```

Comment:
Gotcha.  In the case of a bot it might be worth it to consider updating.  It would allow us to use the full set of ES6 features in the code we write.